Source/ImPlotWidgets/Private/LineFilledGraphWidget.cpp

```diff
@@ -64,17 +91,22 @@ void SFilledGraphWidget::OnImGuiDraw() const
 		ImPlot::PushStyleVar(ImPlotStyleVar_PlotPadding, ImVec2(20, 20));
 		ImPlot::PushStyleVar(ImPlotStyleVar_LabelPadding, ImVec2(10, 10));
 
-		ImPlot::SetNextPlotLimits(0, XMaxValue, 0, YMaxValue*1.1, ImGuiCond_Always);
+		ImPlot::SetNextPlotLimits(0, XMaxValue, YMinValue*1.1, YMaxValue*1.1, ImGuiCond_Always);
```

Comment:
```suggestion
		ImPlot::SetNextPlotLimits(0, XMaxValue, YMinValue * 1.1f, YMaxValue * 1.1f, ImGuiCond_Always);
```

Ymin / XMin are floats, so you want to be explicitly stating that the multipliers are floats too.